### DISCUSSION TITLE
Are menu navigation keys hard coded? If so, in which file?

### DISCUSSION THREAD

**Discussion by vide0hanz** [Novum/vkQuake#554] [Ideas]
I'm wondering if there is an easy-ish way to change the menu navigation keys to something other than the arrow, escape, and return/enter keys. For example, navigating the menus with WASD or HJKL. Are nav keys exposed via console command like other keybinds or are these hard coded?

**Comment by sto6** [comment]
Related: Is there a way to disable menu-nav-by-mouse?

When 1st coming to vkQuake from other-engine, surprised that one can navigate menus using the mouse. I still sometimes load the wrong save-game because, despite arrowing to the correct one, I'd slightly move the mouse (which didn't affect menu-nav in other-engine).

EDIT: Made a little worse, since (macos) on initial entry to the load menu, the mouse-cursor is often at the center of screen, or near a far edge of screen, most definitely not aligned vertically to the row-marker on the save-game to load, and m-cursor does not track/follow when arrow up/down moves the marker: So any small mouse movement, even just left/right, jumps the marker directly. Those row-change by mouse are also silent, where arrow-based nav has sound effects, adding to the initial bewilderment.

"You may find yourself, living in a shotgun shack ... And you may ask yourself, Well... how did I get here?"
